src/main/java/Deadline.java

```diff
@@ -0,0 +1,13 @@
+public class Deadline extends Task{
+    public Deadline(String description) {
+        super(description);
+    }
+
+    private String date = this.description.split("/by")[1];
```

Comment:
Try to avoid magic literals. There has been a few more instances in the rest of your code. You can use a named constant makes the code easier to understand because the name tells us more about the meaning of the number.